low-level/framework/rabbitmq/__init__.py

```diff
@@ -13,4 +13,10 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-"""Modules for handling rabbitMQ processing"""
\ No newline at end of file
+"""Modules for handling rabbitMQ processing"""
```

Comment:
why file under rabbitmq folder? Renaming taking care through separate PR i guess

Reply:
We have separate (sub)task for removing rabbitmq references. That's why  I am not changing anything related to that in this PR. This PR is just for integrating message bus functionality.